becca-new-app/src/App.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-
+const todoList = [
+  { id: 1, title: 'Read over assignments'},
+  { id: 2, title: 'Exercise for 1 hour'},
+  { id: 3, title:  'Make dinner'}
+];
 function App() {
-  return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+return (
+<div>
+<h1>Todo List</h1>
+<ul>
+  {todoList.map(function(todoItem){
```

Comment:
Convert this to an arrow function in the next assignment. We usually use arrow functions when mapping things in React.